grakn-client/src/main/java/ai/grakn/remote/RemoteGraknTx.java

```diff
@@ -82,11 +84,11 @@ private RemoteGraknTx(GraknSession session, GraknTxType txType, GrpcClient clien
         this.client = client;
     }
 
-    static RemoteGraknTx create(RemoteGraknSession session, GraknTxType txType) {
+    public static RemoteGraknTx create(RemoteGraknSession session, TxRequest openRequest) {
```

Comment:
Maybe a TODO about saying this should accept an interface? Or not pass in the session at all.